test/Npgsql.OpenTelemetry.Tests/Npgsql.OpenTelemetry.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+    <PropertyGroup>
+        <SignAssembly>false</SignAssembly>
```

Comment:
This can move to test/Directory.Build.props

Reply:
It seems some of the test projects need to be signed, since they have `InternalsVisibleTo` them from a signed assembly